docs/source/resources/index.rst

```diff
@@ -0,0 +1,92 @@
+Resources
+=========
+
+.. contents:: |toctitle|
+    :local:
+
+Background
+----------
+
+Differential privacy: A primer for a non-technical audience
+
+https://salil.seas.harvard.edu/publications/differential-privacy-primer-non-technical-audience
```

Comment:
I personally think this would work a lot better if the text were links.

Reply:
Yeah, me too. Maybe just simple links for now. Just link the title, I mean.

Reply:
Yeah making the titles into links would be good.